finetune_openai_models/README.md

```diff
@@ -16,3 +16,23 @@ Create a file called `Config.toml` at the root of the project.
 ```
 openAIToken="<OPENAI_API_KEY>"
 ```
+
+### Preparing training data jsonl file
```

Comment:
We can add markdown link to the `prepare training data` section instead of giving the link separately

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/58/commits/7ed3aaa7d8d758e1ca7b378d74f1495f22553c07.